src/components/controls/miscInfoText.js

```diff
@@ -64,3 +64,11 @@ export const ExplodeTreeInfo = (
     It works best when the trait doesn&apos;t change value too frequently.
   </>
 );
+
+export const ToggleFocusInfo = (
```

Comment:
[commenting on this line to enable a thread, it's not related to the info content]

Animation + focus gets a little crazy. I don't think it's worth resolving this at the moment - it's labelled "experimental" after all - but wanted to note it here.